src/Bridge/Symfony/Bundle/Resources/public/logo-header.svg

```diff
@@ -1,100 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do you delete this file? It is still referenced in `src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig`.